### ISSUE TITLE
[a11y] id attribute value must be unique

### ISSUE THREAD

**Issue by pdehaan** [mozilla/fxa#12916]
## Description

axe devtools reports a bunch of repeated `id` attributes on `<div>` elements for **Connected Services** section.

### Steps to reproduce

1. Go to https://accounts.stage.mozaws.net/settings
2. Run axe add-on via DevTools

### Expected result

`id` attributes should be unique.

### Actual result

> Ensures every id attribute value is unique

The "Connected Services" section has a bunch. of entries like this:

```html
<div class="my-1" id="service" data-testid="settings-connected-service" data-name="Chrome 101, macOS 10.15.7">...</div>
<div class="my-1" id="service" data-testid="settings-connected-service" data-name="Firefox 102, macOS 10.15">...</div>
```

### Environment

macOS; axe add-on

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXA-5110)

**Comment by data-sync-user** [comment]
➤ Meghan Sardesai commented:

Setting Story Points to a 3 because the id is attached to oauth checks for appending and removing services, requiring additional testing.
